modules/openapi/component-protocol/scenarios/cmp-dashboard/components/cpuTable/model.go

```diff
@@ -0,0 +1,105 @@
+package cpuTable
+
+import (
+	protocol "github.com/erda-project/erda/modules/openapi/component-protocol"
+	"github.com/erda-project/erda/modules/openapi/component-protocol/scenarios/cmp-dashboard/common"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type CpuInfoTable struct {
+	CtxBdl     protocol.ContextBundle
+	Type       string                 `json:"type"`
+	Props      map[string]interface{} `json:"props"`
+	Operations map[string]interface{} `json:"operations"`
+	State      common.State           `json:"state"`
+	Data       []RowItem              `json:"data"`
+}
+type Columns struct {
+	Title     string `json:"title"`
+	DataIndex string `json:"dataIndex"`
+	Width     int    `json:"width,omitempty"`
+	Sortable  bool   `json:"sorter"`
+}
+type Meta struct {
+	Id       int `json:"id,omitempty"`
+	PageSize int `json:"pageSize,omitempty"`
+	PageNo   int `json:"pageNo,omitempty"`
+	// todo :
+	SortColumn string `json:"sorter"`
+}
+
+type RowItem struct {
```

Comment:
table component struct already define in framework code, is it a table component?

Reply:
modules/openapi/component-protocol/component_spec/table/spec.go